exercises/ruby-1-b/ruby_1_b.rb

```diff
@@ -1,7 +1,7 @@
 class Library
 
   # Title;ISBN;Author;Published
-  BOOKS = <<~DATA
+  CATALOG_DATA = <<~DATA
```

Comment:
Is `_DATA` redundant here?  Any catalog I have ever seen contains data, so that could be assumed.

Reply:
I think that `Catalog` implies that it's an actual catalog, but it's not - it's a CSV. And because I'd rather not encourage the use of the CSV library, data feels like a more agnostic choice. I'm happy for an alternative word, or even for this just to be called `DATA`, but I think we shouldn't imply that it's a rich-object.